tel-search-connector-demo/src_hd/com/axonivy/connector/telsearch/demo/AddressFinder/AddressFinder.xhtml

```diff
@@ -0,0 +1,51 @@
+<html xmlns="http://www.w3.org/1999/xhtml"
+	xmlns:f="http://xmlns.jcp.org/jsf/core"
+	xmlns:h="http://xmlns.jcp.org/jsf/html"
+	xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
+	xmlns:ic="http://ivyteam.ch/jsf/component"
+	xmlns:p="http://primefaces.org/ui"
+	xmlns:pe="http://primefaces.org/ui/extensions">
+<h:body>
+	<ui:composition template="/layouts/frame-9.xhtml">
+		<ui:define name="title">AddressFinder</ui:define>
+		<ui:define name="content">
+
+			<h3>
+				<a href="https://tel.search.ch">tel.search.ch</a> based address
+				finder
+			</h3>
+
+			<h:form id="form">
+				<p:messages />
+				<h:panelGrid columns="3">
+					<p:outputLabel value="Search" for="Search" style="float:left" />
+					<p:inputText value="#{data.search}" id="Search" required="false"
+						style="float:left" />
+
+					<p:commandButton id="Find" actionListener="#{logic.find}"
+						ajax="false" value="Find" update="form" icon="ui-icon-search" />
+					<p:spacer />
+
+				</h:panelGrid>
+				<br />
+				<p:orderList value="#{data.matches}" var="match" responsive="true"
+					style="width:400px" itemValue="#{match}" converter="ivy.ListItem"
+					id="personsList" controlsLocation="right">
+					<f:facet name="caption">Matches</f:facet>
+					<p:column>#{match}</p:column>
+				</p:orderList>
```

Comment:
just a minor, but can we make the entries look sexier by keeping the line breaks
![Selection_1006](https://user-images.githubusercontent.com/15085693/141748258-3e4b9426-57fe-4300-9e3d-8e5ec6b1d244.png)
?

Reply:
Maybe we can build something like this: with columns?
https://www.primefaces.org/showcase/ui/data/orderList.xhtml?jfwid=1bfd1

Reply:
I'd assume that we must just replace '\n' with html output <br/>

Reply:
replacing \n with <br/> does not seem to work. do I need to add anything else?

Reply:
Ok. So I'd leave it as is. Maybe @ivy-lli has a good idea how to build a nicer list output. We can have a look at it tomorrow in Zug.

Reply:
As far as I understand this connector, you can only search for addresses and then they are listed below right?
I think the control's wouldn't be needed in this case right? So I would hide them.

To fix the line breaks, I'm not sure how this work with this widget. I'm not very used to it. But maybe you could simply wrap the match inside a `<pre>` element:
`<p:column><pre>#{match}</pre></p:column>`

Reply:
thanks @ivy-lli with <pre> it works :)